lib/common/asar.js

```diff
@@ -11,55 +11,38 @@
   const cachedArchives = {}
 
   const envNoAsar = process.env.ELECTRON_NO_ASAR && process.type !== 'browser' && process.type !== 'renderer'
-  const isAsarDisabled = function () {
-    return process.noAsar || envNoAsar
-  }
+  const isAsarDisabled = () => process.noAsar || envNoAsar
 
-  const getOrCreateArchive = function (p) {
+  const getOrCreateArchive = p => {
     let archive = cachedArchives[p]
-    if (archive != null) {
-      return archive
-    }
+    if (archive != null) return archive
+
     archive = asar.createArchive(p)
-    if (!archive) {
-      return false
-    }
+    if (!archive) return false
+
     cachedArchives[p] = archive
     return archive
   }
 
   // Clean cache on quit.
-  process.on('exit', function () {
+  process.on('exit', () => {
     for (let p in cachedArchives) {
       if (!hasProp.call(cachedArchives, p)) continue
       cachedArchives[p].destroy()
     }
   })
 
   // Separate asar package's path from full path.
-  const splitPath = function (p) {
+  const splitPath = p => {
```

Comment:
If we're refactoring this can we change this method to return an object with key-values instead of an array where each item at a given index represents a magical thing.

I.e. 
```
{
  didSplit: boolean,
  pathToAsar: string,
  pathInsideAsar: string
}
```